### ISSUE TITLE
Can't discover favicon

### ISSUE THREAD

**Issue by irbisadm** [google/physical-web#823]
Hello!
I'm have favicon issue for this url [http://goo.gl/9c056x](http://goo.gl/9c056x)
physical-web/url-validator/  - shows the icon
Beacon discover in Chrome - shows the icon
Physical web in play services doesn't show the icon.

**Comment by oninross** [comment]
happens to me too sometimes.

**Comment by hayesjordan** [comment]
@irbisadm we were not able to verify this bug. Can you please clarify if you mean the prototype app or nearby?

**Comment by irbisadm** [comment]
@hayesjordan nearby.

**Comment by scottjenson** [comment]
Sorry for the delay. We found the issue with Nearby, they were only letting favions through that were 'big enough' we've finally got a fix through and they should be showing soon. The simplest fix if for you to increase the size of your favicon. The bug filters depending on the resolution of your phone so making it 64x64 is a bit overkill but it should work for now. Like I said, the fix should be rolling out soon.
